content/en/docs/tasks/traffic-management/istioctl-describe/index.md

```diff
@@ -0,0 +1,241 @@
+
+{{< text bash >}}
+$ export RATINGS_POD=$(kubectl get pod -l app=ratings -o jsonpath='{.items[0].metadata.name}')
+$ istioctl experimental describe pod $RATINGS_POD
+{{< /text >}}
+
+The output tells us which containers the pod exposes, the Istio protocol for the microservice on port 9080, and the mutual TLS settings for the pod.
+
+{{< text plain >}}
+Pod: ratings-v1-f745cf57b-qrxl2
+   Pod Ports: 9080 (ratings), 15090 (istio-proxy)
+--------------------
+Service: ratings
+   Port: http 9080/HTTP
+Pilot reports that pod enforces HTTP/mTLS and clients speak HTTP
+{{< /text >}}
+
+## Destination Rules
+
+Next we apply the destination rules suggested by the documentation.  I used mutual TLS,
+so I must apply `destination-rule-all-mtls.yaml`:
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/networking/destination-rule-all-mtls.yaml
+$ istioctl x describe pod $RATINGS_POD
+{{< /text >}}
+
+Applying `destination-rule-all-mtls.yaml` created four destination rules: `details`, `productpage`, `ratings`, and `reviews`.
+
+{{< text bash >}}
+$ istioctl x describe pod $RATINGS_POD
+{{< /text >}}
+
+Now `istioctl x describe` shows additional output:
+
+{{< text plain >}}
+Pod: ratings-v1-f745cf57b-qrxl2
+   Pod Ports: 9080 (ratings), 15090 (istio-proxy)
+--------------------
+Service: ratings
+   Port: http 9080/HTTP
+DestinationRule: ratings for "ratings"
+   Matching subsets: v1
+      (Non-matching subsets v2,v2-mysql,v2-mysql-vm)
+   Traffic Policy TLS Mode: ISTIO_MUTUAL
+Pilot reports that pod enforces HTTP/mTLS and clients speak mTLS
+{{< /text >}}
+
+The destination rule now appears in the output.  This tells us that the `ratings` destination rule is present, and that it defines the subset `v1` which matches this pod.
+Clients talking to the ratings microservice will use mutual TLS.
+
+## Virtual Services
+
+Now I will follow the Bookinfo example to [Request Routing](/docs/tasks/traffic-management/request-routing/) and define some virtual services:
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/networking/virtual-service-all-v1.yaml
+{{< /text >}}
+
+After applying this rule I "describe" the _reviews-v1_ pod:
+
+{{< text bash >}}
+$ export REVIEWS_V1_POD=$(kubectl get pod -l app=reviews,version=v1 -o jsonpath='{.items[0].metadata.name}')
+$ istioctl x describe pod $REVIEWS_V1_POD
+{{< /text >}}
+
+The output resembles what we saw before the virtual services were defined, but we now see that they are present:
+
+{{< text plain >}}
+VirtualService: reviews
+   1 HTTP route(s)
+{{< /text >}}
+
+After applying _virtual-service-all-v1.yaml_ the traffic all goes to version 1.  The "stars disappear".  If this was a real cluster, someone might notice the logs to v2/v3
+are no longer appearing.  Users might notice features and not working.  `describe` will not
+just report the virtual services that configure a pod.  If it seems that a virtual service
+configures a pod, but actually blocks it, the output will include a warning.
```

Comment:
`but actually blocks it` - can you explain what blocks it?  I assume the route is determined but the destination can't respond to the req... in this case it is because unmatching TLS settings?

Reply:
I will reword to `If it seems that a virtual service
configures a pod, but actually blocks traffic by never routing to the pod's subset, the output will include a warning.`